Odata-docs/client/grouping-and-aggregation.md

```diff
@@ -0,0 +1,481 @@
+---
+title: "Grouping and Aggregation in OData Client"
+description: "This tutorial describes how to group and aggregate data in Odata Client"
+
+author: gathogojr
+ms.author: jogathog
+ms.date: 7/25/2022
+ms.topic: article
+ 
+---
+# Grouping and Aggregation in OData Client
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.12.1.md)]
+
+OData Client [v7.12.1](https://www.nuget.org/packages/Microsoft.OData.Client/) introduced support for grouping and aggregation. In this page, we look at the various supported scenarios. The sample data from the 
```

Comment:
Is "Grouping and Aggregation" self-explanatory, or might it be useful to provide a brief explanation of what it's about, maybe something to the effect of "Grouping and aggregation operations are used to summarize data, etc." Not sure if that fits, or whether it's necessary. Just something to consider.

Reply:
@habbes I have added a paragraph describing what grouping and aggregation entails